gui_qt/app.py

```
@@ -4056,17 +4063,85 @@ def _on_sync_backend_changed(self, _index: int) -> None:
+                install_btn.setVisible(False)
+
+    def _litellm_install_running(self) -> bool:
+        process = getattr(self, "_litellm_install_process", None)
+        return process is not None and process.state() == QProcess.ProcessState.Running
+
+    def _on_install_litellm(self) -> None:
+        if self._litellm_install_running():
+            return
+        requirements_path = Path(__file__).resolve().parents[1] / "requirements-litellm.txt"
+        if not requirements_path.is_file():
+            QMessageBox.warning(self, "无法安装 LiteLLM", f"找不到依赖文件：{requirements_path}")
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)